renderer/brave_wallet/brave_wallet_render_frame_observer.cc

```diff
@@ -48,26 +49,30 @@ void BraveWalletRenderFrameObserver::DidCreateScriptContext(
   }
 
   // Wallet provider objects won't be generated for third party iframe
```

Comment:
This comment is outdated because we have `allow=”{solana/ethereum}”` now.
We should also document this and `sandbox=”allow-same-origin”` in https://wallet-docs.brave.com/

Reply:
Comment fixed here:
https://github.com/brave/brave-core/pull/13783/commits/1122ad48727e3d1db86db748370c26575e5c4946#diff-bd0508db5780666e635128717e778f2a51486f055feb9f236d03aefdb1dc122cR48

Working on the wallet documentation out of line from this PR but I will link it here when done.

Reply:
Documentation pull request is here:
https://github.com/brave/brave-wallet-docs/pull/62/files